include/behaviortree_cpp/tree_node.h

```diff
@@ -39,6 +40,7 @@ struct TreeNodeManifest
   std::string registration_ID;
   PortsList ports;
   std::string description;
+  std::vector<ManifestMetadata> metadata;
```

Comment:
As mentioned earlier, I prefer simplicity and to keep it:
```cpp
std::vector<std::pair<std::string, std::string>> metadata;
//or
struct Metadata {
  std::string name;
  std::string value;
};

std::vector<Metadata> metadata;
```

Reply:
Sounds good! Made this change in https://github.com/BehaviorTree/BehaviorTree.CPP/pull/730/commits/96910f265786458ac2f1b33a9f3d1a3adaf26473